src/Compilers/CSharp/Portable/SymbolDisplay/SymbolDisplayVisitor.Members.cs

```diff
@@ -385,15 +384,15 @@ public override void VisitMethod(IMethodSymbol symbol)
                     {
                         containingType = symbol.ReceiverType;
                         includeType = true;
-                        Debug.Assert(containingType != null);
+                        RoslynDebug.Assert(containingType != null);
```

Comment:
>RoslynDebug [](http://example.com/codeflow?start=24&length=11)

Does this change make a difference?

Reply:
➡️ Currently, no. If in the future we have the ability to analyze linked variables (in this case, an effective 'not null when includeType is true'), it would make a difference.

Reply:
> If in the future we have the ability to analyze linked variables (in this case, an effective 'not null when includeType is true'), it would make a difference.

Why that would make a difference for ```RoslynDebug.Assert```, but not for ```Debug.Assert```?

Reply:
`Debug.Assert` is not annotated in all target frameworks, but `RoslynDebug.Assert` is. In general, null checks should always go through `RoslynDebug` so the annotation applies consistently.

Reply:
I am not sure I am buying this. I think `Debug.Assert` works fine for the purpose of nullable analysis in this project, and I would prefer us transitioning in the opposite direction, i.e. stop using ```RoslynDebug```.

Reply:
➡️ Switched all uses of `RoslynDebug` in this PR to `Debug`